slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/qasession/QASession.java

```diff
@@ -1,44 +1,30 @@
 package com.binarystudio.academy.slidez.domain.qasession;
 
-import com.binarystudio.academy.slidez.domain.presentationiteractiveelement.model.PresentationInteractiveElement;
+import com.binarystudio.academy.slidez.domain.iteractiveelement.model.InteractiveElement;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 import org.hibernate.annotations.GenericGenerator;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.OneToOne;
-import javax.persistence.Table;
+import javax.persistence.*;
 import java.util.UUID;
 
-@Entity
-@Data
+@Getter
+@Setter
 @NoArgsConstructor
 @AllArgsConstructor
-@Builder
-@Table(name = "qa_sessions")
-public class QASession {
+@Entity
+@Table(name = "qa_session")
+public class QASession extends InteractiveElement {
 
 	@Id
 	@GeneratedValue(generator = "UUID")
 	@GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
 	@Column(name = "id", updatable = false, nullable = false)
 	private UUID id;
 
-	@OneToOne(cascade = { CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REFRESH })
-	@JoinColumn(name = "presentation_interactive_element_id", referencedColumnName = "id")
-	private PresentationInteractiveElement owner;
-
 	@Column(name = "title")
 	private String title;
 
-	@Column(name = "is_moderated")
-	private boolean isModerated;
```

Comment:
Why have you removed this field?

Reply:
Such field does not exist on instruction I received